langkit/all_metrics.py

```diff
@@ -11,14 +13,14 @@
 from langkit import input_output
 
 
-def init() -> DeclarativeSchema:
-    injections.init()
-    topics.init()
-    regexes.init()
-    sentiment.init()
-    textstat.init()
-    themes.init()
-    toxicity.init()
-    input_output.init()
+def init(config: Optional[LangKitConfig] = None) -> DeclarativeSchema:
```

Comment:
Schema factory functions should take all the schema arguments.

Reply:
Maybe. I think this wrapper of metrics represents something like a preset of metric collection. It is ok if this is a simplification over the degrees of freedom in generating a declarative schema.

Reply:
Then I'd separate the concerns... have the collective `init()`s just initialize the modules and use `udf_schema()` to get the schema.